modules/retail/pages/retail-offline.adoc

```diff
@@ -30,10 +33,12 @@ SALT_TIMEOUT = 60
 For more about kernel parameters, see xref:retail:retail-formula-pxe.adoc[].
 
 
+
 == Offline Terminal Updates
 
 If there is low bandwidth or an unstable connection between the branch server and the terminal, images can be cached in advance and the upgrade performed on the terminals later on.
 
+// This looks incomplete or not detailed enough
```

Comment:
Might be good to raise an issue on this, so we can do the research

Reply:
add to the description here: https://github.com/SUSE/spacewalk/issues/8364 --is this the right place?